src/main/java/de/retest/recheck/ignore/Filter.java

```diff
@@ -0,0 +1,62 @@
+package de.retest.recheck.ignore;
+
+import de.retest.recheck.ui.descriptors.Element;
+import de.retest.recheck.ui.descriptors.IdentifyingAttributes;
+import de.retest.recheck.ui.diff.AttributeDifference;
+
+/**
+ * General interface to ignore changes during Diffing
+ *
+ * In principal, we create a diff for everything that differs, and put that into the result file. Then it is up to the
+ * UI (GUI or CLI) to hide ignored differences – this is what this interface is for. This is pretty much inline with how
+ * Git works – the diff is there, it just doesn't show up when ignored.
+ *
+ * For more details, see <a href="https://github.com/retest/recheck/wiki/How-Ignore-works-in-recheck" target="_top">the
```

Comment:
What about the url, should i leaf it unchanged or what are the plans here?
CC @beatngu13 , @roesslerj